test/lib/ufe/testParentCmd.py

```diff
@@ -366,11 +351,6 @@ def testAlreadyChild(self):
             stage = mayaUsd.ufe.getStage(str(shapeSegment))
 
             # check GetLayerStack behavior
-            self.assertEqual(stage.GetLayerStack()[0], stage.GetSessionLayer())
-            self.assertEqual(stage.GetEditTarget().GetLayer(), stage.GetSessionLayer())
-
-            # set the edit target to the root layer
-            stage.SetEditTarget(stage.GetRootLayer())
             self.assertEqual(stage.GetEditTarget().GetLayer(), stage.GetRootLayer())
```

Comment:
@kxl-adsk merged in dev into my branch and fixed the parenting test failures accordingly.